benchmark/_http-benchmarkers.js

```diff
@@ -15,7 +15,8 @@ class AutocannonBenchmarker {
     this.name = 'autocannon';
     this.executable =
       process.platform === 'win32' ? 'autocannon.cmd' : 'autocannon';
-    const result = child_process.spawnSync(this.executable, ['-h']);
+    const result =
+      child_process.spawnSync(this.executable, ['-h'], { env: process.env });
```

Comment:
Aren't the `{ env: process.env }` changes unnecessary?

Reply:
@cjihrig in theory yes, for `-h` in here you just want them to run, but I figured that we don't control how these applications execute and for all we know they are linked to (or load) crypto/openssl or some other dependency and might fail even for a simple `-h`. Imagine a custom build of `wrk` that's dynamically linked to a custom library and needs a path in `LD_LIBRARY_PATH`.

I could take these out but it _seems_ appropriate to me to just inherit the environment in all of our external tool exec in the test suite.

Reply:
I totally agree that the tools should inherit the environment. What I meant is that the environment is inherited by default, unless you set values for the `env` option, so `{ env: process.env }` is redundant. I don't mind it either way.

Side note, and possibly a good first contribution: the `child_process` docs could do a better job pointing out that `process.env` is the default. Right now, you have to read a lot of text to see that. IMO, it should be part of the option description, like it is for most of the other options.

Reply:
@cjihrig gotcha, so I've gone overkill here and it's only the single `fork()` that needs the adjustment, will remove these, thanks for pointing this out